### PR TITLE
Fix auto-setting of URL from localized page titles

### PR DESCRIPTION
**Pull request by chillu**
Replace the hardcoded check for english locale ("new-page") with a localized version. Tested in IE8 and Chrome. See https://github.com/silverstripe/silverstripe-translatable/issues/180

### CODE REVIEW

File: tests/model/SiteTreeTest.php [hunk 1]
```diff
@@ -653,7 +653,62 @@ public function testIsSection() {
 		$this->assertTrue($staff->isSection());
 		$this->assertTrue($ceo->isSection());
 	}
-	
+
+	public function testURLSegmentAutoUpdate() {
+		Config::nest();
```
Comment (dhensby) [inline tests/model/SiteTreeTest.php]: Doesn't this need to be unnested at the end of the test?

Comment (chillu) [inline tests/model/SiteTreeTest.php]: Good spotting - was a remnant of the test code split out to `testURLSegmentAutoUpdateLocalized()`. Turns out you can't control `i18n::$current_locale` via the Config API...

**Comment by chillu** [comment]
Rebased and incorporated feedback, @dhensby can you please review again?

**Comment by bueckl** [comment]
Hey guys, any chance this change will make it into the master branch anytime soon?

... would come in handy.  Thanks

**Comment by dhensby** [comment]
Ok, looks good - I'll try to merge it by hand and get rid of all that white spacing you added ;)

**Comment by bueckl** [comment]
Yayyyy, great :)

**Pull request merged**